templates/psf/noleft.html

```diff
@@ -0,0 +1,48 @@
+{# ===== PSF Default Content PAGE ===== #}
+
+{% extends "base.html" %}
```

Comment:
I think extending `templates/psf/default.html` would make the template simpler. You'd only need to override the `content_attributes` and `left_sidebar` template blocks:

```jinja
{# Note that I haven't a chance to test this. #}

{% extends "psf/default.html" %}

{% block content_attributes %}{% endblock %}

{% block left_sidebar %}{% endblock %}
```